enterprise/suppress-user/internal/badgerdb/badgerdb.go

```diff
@@ -333,3 +346,19 @@ func (l blogger) Warningf(fmt string, args ...interface{}) {
 func keyPrefix(workspaceID, userID string) string {
 	return fmt.Sprintf("%s:%s:", workspaceID, userID)
 }
+
+func getMetadataFromBadgerItem(item *badger.Item) (*model.Metadata, error) {
+	itemValue, err := item.ValueCopy(nil)
+	if err != nil {
+		return nil, fmt.Errorf("could not copy item value: %w", err)
+	}
+	var metadata model.Metadata
+	if len(itemValue) == 0 {
+		return nil, nil
+	}
```

Comment:
A previous badgerDB version will have an empty string as a value. In that case we need to return a zero-value metadata, not `nil`
```suggestion
	if len(itemValue) == 0 { // backwards compatibility
		return &metadata, nil
	}
```

Reply:
Done